ipaserver/plugins/groupfas.py

```diff
@@ -109,19 +119,12 @@ def get_fasgroup_attribute(self, entry_attrs, options):
 group.get_fasgroup_attribute = get_fasgroup_attribute
 
 
-def group_add_fas_precb(
-    self, ldap, dn, entry, attrs_list, *keys, **options
-):
+def group_add_fas_precb(self, ldap, dn, entry, attrs_list, *keys, **options):
     """Add fasgroup object class and related attributes.
     """
-    fas_params = [
-        option for option in options
-        if option.startswith("fas") and option != "fasgroup"
-        # fasgroup is a flag, it's always present: filter it out.
-    ]
-    if options.get("fasgroup", False) or fas_params:
-        # add fasgroup object class
-        entry["objectclass"].append("fasgroup")
+    if any(option.startswith("fas") for option in options):
```

Comment:
With only this test, I can't create non-fas groups, because the `fasgroup` flag is always present in options (and set to `False`) when not creating a fas group. That's why I resorted to filtering above in my previous commit. Apparently flags are always in the options dict.